cfalertdialog/src/main/res/values/dimen.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="cfdialog_maxwidth">400dp</dimen>
+
+    <dimen name="cfdialog_outer_margin">10dp</dimen>
+    <dimen name="cfdialog_internal_spacing">10dp</dimen>
```

Comment:
What does internal_spacing mean?

Reply:
Within the alert dialog spacing between the elements(title, message, buttons) is called `internal_spacing`